pkg/xray/resource_xray_security_policy.go

```diff
@@ -85,6 +85,12 @@ func resourceXraySecurityPolicyV2() *schema.Resource {
 										Description:      "The minimum security vulnerability severity that will be impacted by the policy.",
 										ValidateDiagFunc: inList("All Severities", "Critical", "High", "Medium", "Low"),
 									},
+									"fix_version_dependant": {
+										Type:        schema.TypeBool,
+										Optional:    true,
+										Default:     false,
+										Description: "Issues that do not have a fixed version are not generated until a fixed version is available.",
```

Comment:
Include default value?

Reply:
What do you mean? There is a default value, but auto-generation didn't pick it up for some reason

Reply:
In Artifactory provider, we include the default value as part of the description. The auto-gen plugin will copy that description to the doc.

Reply:
added